src/geekbrains/lesson3Practical/Main.java

```diff
@@ -1,9 +1,172 @@
 package geekbrains.lesson3Practical;
 
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
 public class Main {
 
+    static Random rndGenerator = new Random();
+    static Scanner scanUser = new Scanner(System.in);
+
     public static void main(String[] args) {
 
+        Task1();
+        Task2();
+
+    }
+
+    /***
+     * Задание 1.
+     *
+     * Написать программу, которая загадывает случайное число от 0 до 9
+     * и пользователю дается 3 попытки угадать это число.
+     * При каждой попытке компьютер должен сообщить, больше ли указанное пользователем число,
+     * чем загаданное, или меньше.
+     * После победы или проигрыша выводится запрос – «Повторить игру еще раз?
+     * 1 – да / 0 – нет»(1 – повторить, 0 – нет).
+     *
+     */
+    private static void Task1() {
+
+        int guessingNumber = 0;
+
+        do {
+
+            // Загадываем число.
+            int resultNumber = rndGenerator.nextInt(10);
+
+            System.out.printf("\nЗагадано число от 0 до 9.\nПопробуйте угадать, у Вас 3 попытки.\n");
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.